### PR TITLE
Update.cpp

### PR DESCRIPTION
**Pull request by John-ZhangYiqing**
update

### CODE REVIEW

File: mcr.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ bool isWin(char game[3][3]){
 int main(){
 	int i, j;
 	char game[3][3] = {' '}; // Tic-tac-toe
-	char player1 = 'X';
+	char player1 = 'X'; // chess X
```
Comment (hlcheun37-c) [inline mcr.cpp:19]: Duplicate line

**Review by John-ZhangYiqing — COMMENTED** [review]
good

**Review by hlcheun37-c — COMMENTED** [review]
review test

File: mcr.cpp (hunk 1 above)
Comment (hlcheun37-c) [inline mcr.cpp]: Approve

**Review by hlcheun37-c — COMMENTED** [review]
Test review 01

**Review by Jixianzhu3 — COMMENTED** [review]
Comments added to make the lines easier to understand

**Pull request merged**